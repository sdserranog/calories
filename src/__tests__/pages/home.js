import React from 'react'
import { render, cleanup } from '@testing-library/react'
import App from '@pages/home'

afterEach(cleanup)

it('should take a snapshot', () => {
  const { container } = render(<App />)

  expect(container.getElementsByClassName('App').length).toBe(1)
})
