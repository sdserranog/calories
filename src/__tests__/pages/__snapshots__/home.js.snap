// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    Test
  </div>
</DocumentFragment>
`;
